beacon-chain/rpc/beacon/validators.go

```diff
@@ -124,6 +121,9 @@ func (bs *Server) ListValidatorBalances(
 		}
 		balancesCount = len(res)
 	}
+	sort.Slice(res, func(i, j int) bool {
```

Comment:
why do we need to sort here?

Reply:
Results are normally sorted if not using indices and public keys, figured it should be sorted for other requests as well. Wdyt?

Reply:
that sounds good to me!